server/crud/crud_wave_datum.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from sqlalchemy.ext.asyncio import AsyncSession  # type: ignore
+from sqlalchemy.future import select  # type: ignore
+
+from server.crud.base import CRUDBase
+from server.models.wave_datum import WaveDatum
+from server.schemas.wave_datum import WaveDatumCreate, WaveDatumUpdate
+
+
+class CRUDWaveDatum(CRUDBase[WaveDatum, WaveDatumCreate, WaveDatumUpdate]):
+    async def find_one(self, db: AsyncSession, date_recorded: datetime):
+        result = await db.execute(
+            select(WaveDatum).where(WaveDatum.date_recorded == date_recorded)
```

Comment:
You can also refactor this class to take advantage of `self.model`, as noted [here](https://github.com/clairBuoyant/server/pull/44/files#r958396003).